nav2_amcl/CMakeLists.txt

```diff
@@ -68,14 +68,6 @@ set(dependencies
   pluginlib
 )
 
-add_library(motion_models SHARED
```

Comment:
I think we still need a library for use and be installed? It just shouldn't be `target_link_libraries` against the main executables

Reply:
`motion_models` library is same as `motions_lib` , you could find here : https://github.com/ros-planning/navigation2/blob/464bfffec0d81fcc14b4b27ddf30ee83e418436b/nav2_amcl/src/motion_model/CMakeLists.txt#L1-L16

Reply:
Yes,sorry I had just missed that part and created redundant code in CMakelists.txt.